Change Notification's private preview is over, and all users must instead use Monitor's direct integration with Resource Graph. (#2)
CawaMS/EnableChangeNotifications: README.md

# Enable Change Notifications Private Preview - Application Change Analysis

> Thank you for your interest in the Change Notifications private preview from [Application Change Analysis feature in Azure Monitor](https://docs.microsoft.com/en-us/azure/azure-monitor/app/change-analysis). The preview has concluded, and we will be ending support for this implementation of notifications in 2024. We recommend all Azure users to use Azure Monitor's native support for querying the Resource Graph for alert queries on resource change data. See the [annoucement blog](https://techcommunity.microsoft.com/t5/azure-observability-blog/query-azure-resource-graph-from-azure-monitor/ba-p/3918298) and [official documentation](https://learn.microsoft.com/en-us/azure/azure-monitor/logs/azure-monitor-data-explorer-proxy) for more information.

Reviewer: ```suggestion
Thank you for your interest in the Change Notifications private preview from [Azure Monitor's Application Change Analysis feature](https://docs.microsoft.com/en-us/azure/azure-monitor/app/change-analysis). The preview has concluded, and we will be ending support for this implementation of notifications in 2024. We recommend using Azure Monitor's native support for querying the Resource Graph for resource change data alerts. 

For more information, see the [announcement blog](https://techcommunity.microsoft.com/t5/azure-observability-blog/query-azure-resource-graph-from-azure-monitor/ba-p/3918298) and [official documentation](https://learn.microsoft.com/en-us/azure/azure-monitor/logs/azure-monitor-data-explorer-proxy).
```
